tests/e2e/tests/regression/browser/large-key-details-values.e2e.ts

```diff
@@ -0,0 +1,129 @@
+import { rte } from '../../../helpers/constants';
+import { acceptLicenseTermsAndAddDatabaseApi } from '../../../helpers/database';
+import { BrowserPage } from '../../../pageObjects';
+import { commonUrl, ossStandaloneConfig } from '../../../helpers/conf';
+import { deleteStandaloneDatabaseApi } from '../../../helpers/api/api-database';
+import { Common } from '../../../helpers/common';
+import {
+    addHashKeyApi,
+    addListKeyApi,
+    addSetKeyApi,
+    addSortedSetKeyApi,
+    addStreamKeyApi,
+    deleteKeysApi
+} from '../../../helpers/api/api-keys';
+
+const browserPage = new BrowserPage();
+const common = new Common();
+
+const field = common.generateWord(20);
+const value = common.generateSentence(200);
+const value1 = common.generateWord(20);
+const keyNames = [common.generateWord(20), common.generateWord(20), common.generateWord(20), common.generateWord(20), common.generateWord(20)];
+const hashKeyParameters = { keyName: keyNames[0], fields: [{ field, value: value1 }, { field, value }] };
+const streamKeyParameters = { keyName: keyNames[1], entries: [{ id: '*', fields: [[field, value], [field, value1]] }] };
+const setKeyParameters = { keyName: keyNames[2], members: [value] };
+const sortedSetKeyParameters = { keyName: keyNames[3], members: [{ name: value, score: '1' }] };
+const listKeyParameters = { keyName: keyNames[4], element: value };
+
+fixture `Expand/Collapse large values in key details`
+    .meta({ type: 'regression', rte: rte.standalone })
+    .page(commonUrl)
+    .beforeEach(async() => {
+        await acceptLicenseTermsAndAddDatabaseApi(ossStandaloneConfig, ossStandaloneConfig.databaseName);
+        await addHashKeyApi(hashKeyParameters, ossStandaloneConfig);
+        await addStreamKeyApi(streamKeyParameters, ossStandaloneConfig);
+        await addSetKeyApi(setKeyParameters, ossStandaloneConfig);
+        await addSortedSetKeyApi(sortedSetKeyParameters, ossStandaloneConfig);
+        await addListKeyApi(listKeyParameters, ossStandaloneConfig);
+    })
+    .afterEach(async() => {
+        //Delete database
+        await deleteKeysApi(keyNames, ossStandaloneConfig);
+        await deleteStandaloneDatabaseApi(ossStandaloneConfig);
+    });
+test('Verify that user can click on a row to expand it if any of its cells contains a value which is truncated.', async t => {
+    const entryFieldLong = browserPage.streamEntryFields.nth(1).parent(1);
+    const entryFieldSmall = browserPage.streamEntryFields.nth(0).parent(1);
```

Comment:
Can't you use data-id?

Reply:
this is a component common to all virtualized table cells, and for all other tables, it is better not to add data-test-id
this component does not know the context of where it is being drawn

Reply:
Notice it can be break